packages/kg-default-nodes/lib/utils/crop-image-for-email.js

```diff
@@ -0,0 +1,12 @@
+import {isLocalContentImage} from './is-local-content-image';
+
+export const getCroppedImage = (src, options) => {
+    if (isLocalContentImage(src, options.siteUrl) && options.canTransformImage?.(src)) {
+        const [, imagesPath, filename] = src.match(/(.*\/content\/images)\/(.*)/);
+        return {
+            src: `${imagesPath}/size/w256h256/${filename}`,
+            width: 256,
+            height: 256
+        };
+    }
+};
```

Comment:
The naming for this feels very generic for a util that has a specific purpose of resizing to 256x256. That said, is this util function used anywhere? As far as I can see [where we do the resize](https://github.com/TryGhost/Koenig/pull/1453/files#diff-6587e200981fa8375b8137c87242b7adecc0b4802bc5cd1619c002efd0564d88R67-R70) it's done without this function being used.

Reply:
Ah yes, correct, I was meant to remove this util as I realised it's just too specific to be added as a utility.